core/templates/services/svg-sanitizer.service.ts

```diff
@@ -94,15 +106,23 @@ export class SvgSanitizerService {
 
   private _getInvalidSvgTagsAndAttrs(
       svg: Document): {tags: string[], attrs: string[]} {
-    let invalidTags = [];
-    let invalidAttrs = [];
-    let allowedTags = Object.keys(constants.SVG_ATTRS_WHITELIST);
+    let invalidTags: string[] = [];
+    let invalidAttrs: string[] = [];
+
+    // This throws "The type 'readonly ["about", "alignment-baseline, ... 86
+    // more ..., "writing-mode"]' is 'readonly' and cannot be assigned to the
+    // mutable type 'string[]'." We need to suppress this error because of
+    // stricter type checking.
+    // @ts-ignore
+    let SvgAttrsWhitelist: Record<string, string[]> =
+     constants.SVG_ATTRS_WHITELIST;
```

Comment:
Indent by 2 on continuation lines. Consider surroudning with parens too, e.g. 

```
    let svgAttrsWhitelist: Record<string, string[]> = (
      constants.SVG_ATTRS_WHITELIST);
```

Also, shouldn't svgAttrsWhitelist start with a lowercase letter?

Finally, please rename whitelist to allowlist, we prefer using that terminology instead.

Reply:
done